src/main/kotlin/io/github/inductiveautomation/kindling/utils/TabStrip.kt

```diff
@@ -151,6 +177,32 @@ open class TabStrip : DnDTabbedPane() {
         )
     }
 
+    fun editTabTitle(tabIndex: Int) {
+        val textField = JTextField(
+            getTitleAt(tabIndex),
+        )
+
+        textField.addKeyListener(
+            object : KeyAdapter() {
+                override fun keyPressed(e: KeyEvent?) {
```

Comment:
Minor: You can assert that `e` is not null here (it won't be, but Kotlin is being defensive)